tests/scripts/embedding_log_test.py

```diff
@@ -4,6 +4,7 @@
 import sys
 
 import ansys.mechanical.core as mech
+from ansys.mechanical.core.embedding.config import Configuration as EmbeddingConfiguration
```

Comment:
@RobPasMue @klmcadams I'm not too happy about two classes that are both called Configuration.  Any suggestions for renaming them?

Reply:
I'd try to avoid classes with the same nomenclature inside the same library. Better go with ``EmbeddingConfiguration`` and ``RemoteConifguration``.

Reply:
@RobPasMue ok, we can do something like that.  I think the names should be `LoggingConfiguration` and `AddinConfiguration` since its not related to embedding/remote.  I don't know why I aliased it as `EmbeddingConfiguration` here.

But this is a breaking change (esp for logging configuration, which might already be used).  How should we do that?

Reply:
@RobPasMue client side!

Reply:
@RobPasMue I changed the name of the addin configuration class without any deprecation and created an issue for the logging configuration